### PR TITLE
Niloofar Sadeghi / Refactor who-we-are page

### PR DESCRIPTION
**Pull request by niloo-fs**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/who-we-are/_data.tsx [hunk 1]
```diff
@@ -1,57 +1,65 @@
 import React, { ReactElement } from 'react'
-import { Localize } from 'components/localization'
+import { Localize, localize } from 'components/localization'
 import Shield from 'images/svg/who-we-are/shield.svg'
 import Star from 'images/svg/who-we-are/star.svg'
 import People from 'images/svg/who-we-are/people.svg'
 import Hands from 'images/svg/who-we-are/hands.svg'
 
 export const first_section_texts = [
```
Comment (suthesan-deriv) [inline src/pages/who-we-are/_data.tsx]: Should we type this too?

File: src/pages/who-we-are/_data.tsx [hunk 2]
```diff
@@ -187,81 +214,110 @@ export const desktop_pins: MapPinType[] = [
...
     {
+        id: 13,
         left: '43%',
         top: '26.9%',
         title: <Localize translate_text="Guernsey" />,
         link: '/careers/locations/guernsey',
     },
 ]
+
+export const our_offices_count = [
```
Comment (suthesan-deriv) [inline src/pages/who-we-are/_data.tsx:269]: Same here

File: src/pages/who-we-are/_data.tsx [hunk 3]
```diff
@@ -187,81 +214,110 @@ export const desktop_pins: MapPinType[] = [
...
+    { id: 1, count: localize('750+'), title: localize('employees') },
+    { id: 2, count: localize('50+'), title: localize('nationalities') },
+    { id: 3, count: localize('13'), title: localize('locations') },
+    { id: 4, count: localize('10'), title: localize('countries') },
+]
+
+export const deriv_numbers = [
```
Comment (suthesan-deriv) [inline src/pages/who-we-are/_data.tsx:276]: Same here

File: src/pages/who-we-are/_MakeTrading.tsx [hunk 4]
```diff
@@ -38,9 +38,9 @@ const StyledFirstSectionText = styled(Header)`
 const MakeTrading = () => {
     return (
         <FirstSectionContainer padding="120px 0 80px" background="var(--color-white)">
-            {first_section_texts.map((item, index) => (
-                <StyledFirstSectionText as="p" type="unset" key={index} size="24px">
-                    {item.text}
+            {first_section_texts.map(({ text }) => (
+                <StyledFirstSectionText as="p" type="unset" key={+text} size="24px">
```
Comment (sean-binary) [inline src/pages/who-we-are/_MakeTrading.tsx:42]: Doesn't this return NAN if we try to coerce a text string to a number?

File: src/pages/who-we-are/_OurOffices.tsx [hunk 5]
```diff
@@ -176,56 +109,28 @@ const OurOffices = () => {
...
                     <MapImage data={data['earth']}>
                         {desktop_pins.map((pin) => (
-                            <MapPin
-                                key={pin.title}
-                                left={pin.left}
-                                top={pin.top}
-                                title={pin.title}
-                                link={pin.link}
-                            />
+                            <MapPin key={+pin.title} {...pin} />
```
Comment (sean-binary) [inline src/pages/who-we-are/_OurOffices.tsx:112]: Same here and a few other instances below as well.

File: src/pages/who-we-are/_MakeTrading.tsx (hunk 4 above)
Comment (niloo-fs) [reply src/pages/who-we-are/_MakeTrading.tsx:42]: oh yes that's right

Comment (niloo-fs) [reply src/pages/who-we-are/_MakeTrading.tsx:42]: I didn't want to pass index as key

**Review by sean-binary — DISMISSED** [review]

**Review by sean-binary — APPROVED** [review]

**Review by sean-binary — APPROVED** [review]
LGTM. Thanks @niloo-fs !

**Pull request merged**